src/main/java/service/operation/ReturnOperation.java

```diff
@@ -0,0 +1,23 @@
+package service.operation;
+
+import dao.FruitDao;
+import dao.FruitDaoImpl;
+
+public class ReturnOperation implements OperationHandler {
+    private FruitDao fruitDao;
+
+    public ReturnOperation() {
+        this.fruitDao = new FruitDaoImpl();
```

Comment:
Don't instantiate a specific implementation of `FruitDao` inside the `ReturnOperation` class. This violates the Dependency Inversion principle, which is part of SOLID. Instead, pass the `FruitDao` implementation through the constructor to make your service independent.